sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -355,6 +355,16 @@ object SQLConf {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefault(64 * 1024 * 1024)
 
+
+  val SHUFFLE_FETCH_CONTINUOUS_BLOCKS_IN_BATCH_ENABLED =
+    buildConf("spark.sql.adaptive.shuffle.fetchContinuousBlocksInBatch.enabled")
```

Comment:
instead of creating the "shuffle" namespace, how about `spark.sql.adaptive.fetchShuffleBlocksInBatch.enabled`?

Reply:
Sure, done in afd74e5.
There's already has `spark.sql.adaptive.shuffle` namespace.